<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang=
"en-gb">
 <head>
  <title>Bioinformatics Web Servers - University of Reading</title>
  <meta name="description" content="" />
  <meta name="keywords" content="" />
  <meta name="Author" content="The University of Reading" />
  <meta http-equiv="content-language" content="en-gb" />
  <meta http-equiv="content-style-type" content="text/css" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <style type="text/css">
  @import "https://www.reading.ac.uk/web/Templates/standard-school_files/school.css";
  @import "https://www.reading.ac.uk/web/Templates/standard-school_files/print.css";
  /*reference your own unit title image in the rule below*/
  div#header h2 {background:url(../bioinf.gif) no-repeat}
  </style>
  <!--link to the stylesheet corresponding to your colour choce below - see the accompanying documentation-->
  <link rel="stylesheet" type="text/css" href="https://www.reading.ac.uk/web/StyleSheets/Reading_navy.css" />
  <!--[if lte IE 7]>
  <link rel="stylesheet" type="text/css" href="https://www.reading.ac.uk/web/templates/standard-school_files/school_ie.css" />
 <![endif]-->
 </head>
 <body>
  <div id="wrapper">
   <div id="header">
     <ul class="hidden">
     <li>
      <a href="https://www.reading.ac.uk/about/about-accessibility.asp" accesskey="0">0 -
      Access key details</a>
     </li>
     <li>
      <a href="https://www.reading.ac.uk/about/about-faqs.asp" accesskey="5">5 - Frequently
      Asked questions (FAQ)</a>
     </li>
     <li>
      <a href="https://www.reading.ac.uk/about/about-contacts.asp" accesskey="9">9 - Main
      University of Reading contacts</a>
     </li>
    </ul>
    <h1>
     <a href="https://www.reading.ac.uk/">University of Reading</a>
    </h1>
    <h2><!--Unit Title goes here - replaced by image via CSS-->
     HTML Templates
    </h2>
   </div>
   <h1 class="hidden"> Accessibility navigation </h1>
   <div id="lower">
    <div id="container2">
     <hr id="line" />
     <div id="content">
      <div id="centerColumn">
       <!--centre column content starts here-->
	   <h2>
        The IntFOLD Integrated Protein Structure and Function Prediction Server
       </h2>
		<h3>About the server</h3>
		<p>The IntFOLD server provides a unified interface for:
		<ol>
		<li>Tertiary structure prediction/3D modelling</li>
		<li>3D model quality estimates - with the option to refine/fix errors</li>
		<li>Intrinsic disorder prediction</li>
		<li>Domain prediction</li>
		<li>Prediction of protein-ligand binding residues</li>
		</ol>
		<p><b>This website is free and open to all users and there is no login requirement.</b>
		<br>
        <a href=https://www.reading.ac.uk/about/privacy>Privacy and cookies</a>
        <br></p>
		<p>Fair usage policy: 
                <p>You are only permitted to have 1 job running at a time for each IP address, so please wait until your previous job completes before submitting further data. If you already have a job running then you will be notified and your uploaded data will be deleted. Once your job has completed your IP address will be unlocked and you will be able to submit new data. 
		
		<h3><a HREF="IntFOLD7_form.html">IntFOLD7 submission form</a></h3>
		(latest version, see below for older versions)
		</p>
		<h3><a HREF="IntFOLD_help.html">Help page</a></h3>
		<h3><a HREF="IntFOLD_help.html#examples">Sample output</a></h3>
		<h3>Latest server references</h3>
		<ul>
		<li>McGuffin, L. J., Edmunds N. S., Genc, A. G., Alharbi, S. M. A., Salehe, B. R. and Adiyaman, R. (2023) Prediction of protein structures, functions and interactions using the IntFOLD7, MultiFOLD and ModFOLDdock servers. Nucleic Acids Research. gkad297. <a href="https://doi.org/10.1093/nar/gkad297">DOI</a> <a href="https://pubmed.ncbi.nlm.nih.gov/37102670/">PubMed</a></li>    
		<li>McGuffin, L.J., Adiyaman, R., Maghrabi, A.H.A., Shuid, A.N., Brackenridge, D.A., Nealon, J.O. & Philomina, L.S. (2019) IntFOLD: an integrated web resource for high performance protein structure and function prediction. Nucleic Acids Research, 47, W408-W413. <a href="https://doi.org/10.1093/nar/gkz322">DOI</a> <a href="https://www.ncbi.nlm.nih.gov/pubmed/31045208">PubMed</a></li>
	     </ul>
		<h3>Paper in CASP12 proceedings</h3>
		<ul>
		<li>McGuffin, L.J., Shuid, A.M., Kempster, R., Maghrabi, A.H.A., Nealon  J.O., Salehe, B.R., Atkins, J.D. & Roche, D.B. (2017) Accurate Template Based Modelling in CASP12 using the IntFOLD4-TS, ModFOLD6 and ReFOLD methods. Proteins: Structure, Function, and Bioinformatics, 86 Suppl 1, 335-344, doi: 10.1002/prot.25360. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28748648">PubMed</a></li>		
		</ul>
		
		<h3>News and updates</h3>
		<ul>
		<li>Apr 2023: Paper describing the IntFOLD7, MultiFOLD and ModFOLDdock servers has been accepted by Nucleic Acids Research - <a href="publications.html">Publications</a></li>
        <li>Mar 2023: Docker package for MultiFOLD, MultiFOLD_refine and ModFOLDdock is now available: <a href="https://hub.docker.com/r/mcguffin/multifold">https://hub.docker.com/r/mcguffin/multifold</a></li>
		<li>Dec 2022: Success at CASP and CAMEO: IntFOLD7 is competitive with AlphaFold2 baselines and other new Deep Learning methods in terms of tertiary structure prediction, while also integrating leading model quality estimates, protein-ligand interaction predictions, disorder and domain prediction.</li>
		<li>May 2022: CASP15 prediction season begins! New servers and methods have been developed, which will be independently benchmarked, including: IntFOLD7, FunFOLD4 and ModFOLD9.</li>
        <li>March 2022: new versions of our servers and methods have been registered with CAMEO and are being independently benchmarked, including: IntFOLD7, ModFOLD9 and ModFOLD9_pure.</li> 
        <li>Oct 2021: New server hardware has been installed.</li>
		<li>Aug 2021: Proteins paper accepted - "Modeling SARS-CoV2 proteins in the CASP-commons experiment" - <a href="../publications.html">Publications</a></li>
        <li>Feb 2021: Two post docs are now working on our BBSRC project - <a href="../people/">People</a></li>
        <li>Jan 2021: New hardware installed for IntFOLD component methods.
		<li>Jun 2020: <a href="http://www.reading.ac.uk/news-and-events/releases/PR841761.aspx">Press release</a> for CASP Commons research into SARS-CoV-2 protein structures.</li> 
        <li>May 2020: CASP14 prediction season begins! New servers and methods will be benchmarked.</li>
        <li>Mar-Jun 2020: COVID-19 research - SARS-CoV-2 protein structures have been <a href="https://docs.google.com/spreadsheets/d/1Q66tWjJ1uvYisHBUFSDRWfL7zJJGmz79mb0F3zoArgU/edit?usp=sharing">modelled using IntFOLD6 and scored with ModFOLD8</a> as part of <a href="http://predictioncenter.org/caspcommons/">CASP Commons 2020</a>
        <li>Feb 2020: New hardware has been installed for the new versions of our methods - IntFOLD6 and ModFOLD8 will be benchmarked in CASP14 and CAMEO.</li>
		<li>Apr 2019: IntFOLD5 server paper accepted by Nucleic Acids Research.</li>
		<li>Dec 2018: Success at <a href=http://predictioncenter.org/casp13/index.cgi>CASP13</a> (we are among the top few ranked groups in QA and TBM)! Invited talk & panel contribution given and two posters presented on our new methods and servers. Quoted in <a href="https://www.theguardian.com/science/2018/dec/02/google-deepminds-ai-program-alphafold-predicts-3d-shapes-of-proteins">Guardian article</a> re: CASP13 and DeepMind</li>
        <li>Nov-Dec 2018: user interface improvements and new server hardware.<li>
		<li>May-Aug 2018: CASP13 prediction season.</li>
		<li>May 2018: IntFOLD5 server methods ready for CASP13</li>
		<li>Jul 2017: Template Based Modelling paper accepted for the CASP12 Proteins Special Issue<li>
		<li>Dec 2016: Success at CASP12 (among the top few ranked in TBM and QA)! Two talks and three posters presented on our new methods and servers. The ReFOLD and ModFOLD6 servers are now online.</li>
		<li>Feb 2015: <a href="http://www.sciencemag.org/content/347/6225/1014.abstract">Science paper</a> published containing results from the IntFOLD server. <a href="https://www.reading.ac.uk/news-and-events/releases/PR624451.aspx"> Press release.</a></li>
		<li>Dec 2014: new interactive model visualisations implemented using the JSmol/HTML5 framework (also works on tablets and phones).<li>
		<li>Dec 2014: improved job status notification system implemented.
		<li>Dec 2014: Success at CASP11, especially in the <a href="http://www.predictioncenter.org/casp11/doc/presentations/CASP11_QA_AK.pdf">Quality Assessment</a> (QA3) category
		<li>Aug 2014: CASP11 prediction season complete.
		<li>May 2014: CASP11 starts! New versions of the server component methods will be benchmarked. Provision of new CASP & CAMEO structural and functional data types/formats.
		<li>Sep 2013: IntFOLD2 integrated with the <a href="http://proteinmodelportal.org">Protein Model Portal</a></li>
		<li>Dec 2012: success at CASP10 - short talks given on quality assessment and function prediction
		<li>Dec 2012: paper reporting extensive application of servers has been published in BMC Genomics
		<li>Aug 2012: IntFOLD2 open beta version online for testing please send feedback to <a href=mailto:l.j.mcguffin@reading.ac.uk>l.j.mcguffin@reading.ac.uk</a></li>
		<li>May 2012: Multi-Template Modelling (MTM) method Bioinformatics paper published (Buenavista et al., Bioinformatics, 2012), the method forms the basis of IntFOLD2-TS predictions</li>
		<li>Sept 2011: Confidence scores (p-values) now included in graphical output. <a href="https://www.reading.ac.uk/bioinf/IntFOLD/IntFOLD_help.html#TS">Help page</a></li>
		<li>Aug 2011: Interactive IntFOLD-TS output now includes model-template superpositions - <a href="https://www.reading.ac.uk/bioinf/servlets/nFOLD/IntFOLDResultsFileServlet?time=19_1_49_576_1-12-2010&target_name=1kjjakib581aj2tt&modelfilename=19_1_49_576_1-12-2010_1ny5A_HHsearch_TS1.bfact#Coverage">Example</a></li>
		<li>May 2011: IntFOLD-TS method paper now in press (CASP9 special issue) - <a href="../publications.html">Publications</a></li>
		<li>May 2011: FunFOLD paper now in press - <a href="../publications.html">Publications</a> <a href="../downloads/">Download FunFOLD</a> </li>
		<li>March 2011: IntFOLD server paper now in press (NAR Web Server issue) - <a href="../publications.html">Publications</a></li>
		<li>March 2011: Methods paper describing application of servers to Blumeria proteome now in press - <a href="../publications.html">Publications</a></li>
		<li>Dec 2010: The IntFOLD server version is now out of beta.
		<li>Dec 2010: The IntFOLD-TS method was the focus of our invited talk for the CASP9 methods session. <a href="Methods_McGuffin.pptx">Download slides (.pptx).</a>
		</ul>
		<h3>Old server references:<br></h3>
		<ul>
	        <li>McGuffin, L.J., Atkins, J., Salehe, B.R., Shuid, A.N. & Roche, D.B. (2015) IntFOLD: an integrated server for modelling protein structures and functions from amino acid sequences. Nucleic Acids Research, 43, W169-73. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25820431">PubMed</a></li>
		<li>Roche, D. B., Buenavista, M. T., Tetchner, S. J. &amp; McGuffin, L. J. (2011) The IntFOLD server: an integrated web resource for protein fold recognition, 3D model quality assessment, intrinsic disorder prediction, domain prediction and ligand binding site prediction. Nucleic Acids Res., 39, W171-6. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21459847">PubMed</a></li>
		</ul>
		<BR>
		<h3>The IntFOLD component standalone methods<br></h3>
		<b>IntFOLD2-TS</b>
		<li>Buenavista, M. T., Roche, D. B., &amp; <b>McGuffin, L. J.</b> (2012) Improvement of 3D protein models using multiple templates guided by single-template model quality assessment. Bioinformatics, 28, 1851-1857. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22592378">PubMed</a></li>
		<b>IntFOLD-TS (original nFOLD4 version):</b>
		<li>McGuffin, L. J. &amp; Roche, D. B. (2011) Automated tertiary structure prediction with accurate local model quality assessment using the IntFOLD-TS method. Proteins: Structure, Function, and Bioinformatics, 79 Suppl 10, 137-46. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22069035">PubMed</a></li>
		<b>IntFOLD-FN (FunFOLD):</b>
		<li>Roche, D. B., Tetchner, S. J. &amp McGuffin, L. J. (2011) FunFOLD: an improved automated method for the prediction of ligand binding residues using 3D models of proteins. BMC Bioinformatics, 12, 160. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21575183">PubMed</a></li>
		<b>IntFOLD-QA (ModFOLD):</b>
		<li>McGuffin, L. J. &amp; Roche, D. B. (2010) Rapid model quality assessment for protein structure predictions using the comparison of multiple models without structural alignments. Bioinformatics, 26, 182-188. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19897565">PubMed</a></li>
		<b>IntFOLD-DR (DISOclust):</b>
		<li>McGuffin, L. J. (2008) Intrinsic disorder prediction from the analysis of multiple protein fold recognition models. Bioinformatics, 24, 1798-804. <a href=http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=18579567>PubMed</a></li>
		</ul>
		<h3>Older versions</h3>
		<h3><a HREF="IntFOLD6_form.html">IntFOLD6 submission form</a></h3>
		<h3><a HREF="IntFOLD5_form.html">IntFOLD5 submission form</a></h3>
		<h3>IntFOLD4</h3>
		<h3>IntFOLD3</h3>
		<h3>IntFOLD2</h3>
		<h3>IntFOLD(original version)</h3></ul>
		<!--<h3><a HREF="IntFOLD4_form.html">IntFOLD4 submission form</a></h3>
		<h3><a HREF="IntFOLD3_form.html">IntFOLD3 submission form</a></h3>
		<h3><a HREF="IntFOLD2_form.html">IntFOLD2 submission form</a></h3>
		<h3><a HREF="IntFOLD_form.html">IntFOLD submission form (original version)</a></h3></ul>-->
                <h3>Case study</h3>
		The animation below shows the top IntFOLD model for CSRP3 from mouse, which serves to demonstrate results from each of the integrated methods. Two globular zinc binding domains are linked by an intrinsically disordered region. The model helps us to understand how this protein might act as a stress sensor in cardiac myocytes. Click on the image to view the full IntFOLD results.
		<br><br>
		<!--<a href="https://www.reading.ac.uk/bioinf/servlets/nFOLD/IntFOLDresults.jsp?time=11_0_32_764_11-2-2011_All&md5=t45e4h6ogf4n8l8l&targetname=cysteineandglycine-richprotein3" ><img border="0" src="CSRP3_IntFOLD.gif"></a>-->
		<a href="https://www.reading.ac.uk/bioinf/servlets/nFOLD/IntFOLD3results.jsp?time=15_30_2_462_20-12-2014_All&md5=t45e4h6ogf4n8l8l&targetname=CSRP3" ><img border="0" src="CSRP3_IntFOLD.gif"></a>
		
		
		

<!--centre column content ends here-->
      </div>
     </div>
     <div id="rightColumn">
	 <!--<div id="internal">
       <a href="/internal/Testing/">For current students<br />
       and staff</a>
      </div>-->
       <div class="panel">
         <div class="inner">
           <h2>
         Contact
        </h2>
        <p>
         Tel: 0118 378 6332
        </p>
        <p>
         Email: <a href=
         "mailto:l.j.mcguffin@reading.ac.uk">l.j.mcguffin<br>@reading.ac.uk</a>
        </p>
        <ul>
         <li>
	  <a href="../contact.html">Full contact details</a>
         </li>
        </ul>
          <a href="https://twitter.com/IntFOLD?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @IntFOLD</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
         </div>
       </div>
     </div>
    </div>
   </div>
   <div id="leftColumn">
    <h1 class="hidden">
     Page navigation
    </h1><!--local navigation-->
    <ul id="subNav">
     <li class="back">
      <a href="/" accesskey="1"><span title="University of Reading">UoR Home</span></a>
     </li>
     <li>
      <a href="../index.html">Bioinformatics Servers Home</a>
     </li>
    </ul>
    <ul id='ae_menu'>
     <li>
      <a href='../MultiFOLD/'>MultiFOLD</a>
     </li>
     <li>
      <a href='../ModFOLDdock/'>ModFOLDdock</a>
     </li>
     <li>
      <a href='../IntFOLD/'>IntFOLD</a>
     </li>
     <li>
      <a href='../ModFOLD/'>ModFOLD</a>
     </li>
     <li>
      <a href='../ReFOLD/'>ReFOLD</a>
     </li>
     <li>
      <a href='../FunFOLD/'>FunFOLD</a>
     </li>     
     <li>
      <a href='../DISOclust/'>DISOclust</a>
     </li>
     <li>
      <a href='../DomFOLD/'>DomFOLD</a>
     </li>
     <li>
      <a href='../nFOLD/'>nFOLD3</a>
     </li>     
    </ul><!--local navigation END-->
    <!--<div class="panel">
     <div class="inner">
	 <h2>
       See Also
      </h2>
      <ul>
       <li>Link 1
       </li>
       <li>Link 2
       </li>
      </ul>
	 </div>
    </div>
    <div class="promo" id="promo">
	 <p>
      <strong>Promo Copy</strong> <a href="#">Link to further
      info</a>
     </p>
	</div>-->
   </div>
   <div class="clearing">
    &nbsp;
   </div>
   <div id="footer">
    <h1 class="hidden">
     Footer navigation
    </h1>
   <ul id="footerNav">
     <li>
      <a href="https://www.reading.ac.uk/about/about-accessibility.asp">Accessibility</a>
     </li>
     <li>
      <a href="https://www.reading.ac.uk/about/sitemap.asp">Site Map</a>
     </li>
     <li>      <a href="../contact.html">Contact Us</a>
     </li>
     <li class="last">
      <a href="https://www.reading.ac.uk/about/find/about-findindex.asp">Find us</a>
     </li>
    </ul>
    <p>
     <a href="https://www.reading.ac.uk/about/about-copyright.asp">&copy; University of
     Reading</a>    </p>
   </div>
   <ul id="topNav">
    <li>
     <a href="https://www.reading.ac.uk/life/">Life</a>
    </li>
    <li>
     <a href="https://www.reading.ac.uk/study/">Study</a>
    </li>
    <li>
     <a href="https://www.reading.ac.uk/research/">Research</a>
    </li>
    <li>
     <a href="https://www.reading.ac.uk/working-with-business/">Business</a>
    </li>
    <li>
     <a href="https://www.reading.ac.uk/about/">About</a>
    </li>
    <li>
     <a class="last" href="https://www.reading.ac.uk/a-z/">A-Z</a>
    </li>
   </ul>
   <h1 class="hidden">
    Search Form
   </h1>
   <div id="search">
    <form id="searchForm" action=
    "https://www.reading.ac.uk/search/public/search" method="get"
    name="searchForm">
     <label accesskey="4" for="text">Search</label> <select name=
     "sitesearch" aria-label =âsearch locationâ>
      <option value="">
       University site
      </option>
      <!--enter the url of your site below, see www.reading.ac.uk/cme for an example-->
<option value="[your site URL]">
       This section
      </option>
     </select><input class="text" size="12" maxlength="255" id=
     "text" name="q" /> <input class="submit" type="submit" value=
     "Go" /> <input name="site" value="internal" type="hidden" />
     <input name="output" value="xml_no_dtd" type="hidden" />
     <input name="client" value="internal" type="hidden" />
     <input name="proxystylesheet" value="internal" type=
     "hidden" />
    </form>
   </div>
  </div>
 </body>
</html>
