<!DOCTYPE html>
<html>
<head>
  <!-- Basic -->
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <!-- Mobile Metas -->
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
  <!-- Site Metas -->
  <meta name="keywords" content="" />
  <meta name="description" content="" />
  <meta name="author" content="" />

  <title>Molsoft L.L.C.: Molsoft's Technology</title>
<meta name="keywords" content="technology;structure-based;drug design;homology;modeling;structure prediction;cheminformatics;docking;bioinformatics;software development;refinement;loop;pocket;icmPocketFinder;virtual screening;vls;">
  &nbsp <a href="/index.html"> <img src="/images/MolSoft_Logo2023.jpg" height="60" alt=""> </a>
  &nbsp
  <a href="mailto: info@molsoft.com"> <img src="/images/mail_25.png" height="25" align="middle"  alt=""> </a>
  |
  <a href="https://twitter.com/molsoft"> <img src="/images/twitter25.png" height="25" align="middle"  alt=""> </a>
  <a href="https://www.linkedin.com/company/3634260/admin/"> <img src="/images/linkedin25.png" height="25" align="middle"  alt="">	</a>
  <a href="https://www.youtube.com/channel/UC92_g6Ig0KFnlXOiy5CN1Gg"> <img src="/images/yt25.png" height="25" align="middle"  alt="">	</a>
  |	
  <a href="/new-features.html"> <img src="/images/newfeatures2.png" height="25" align="middle"  alt="">	</a>
 <br>
  <!-- slider stylesheet -->
  <link rel="stylesheet" type="text/css" href="/css/owl.carousel.min.css" />

  <!-- bootstrap core css -->
  <link rel="stylesheet" type="text/css" href="/css/bootstrap.css" />

  <!-- fonts style -->
  <link href="https://fonts.googleapis.com/css?family=Poppins:400,700|Roboto:400,700&display=swap" rel="stylesheet">
  <!-- Custom styles for this template -->
  <link href="/css/style.css" rel="stylesheet" />
  <!-- responsive style -->
  <link href="/css/responsive.css" rel="stylesheet" />
  <script src="/web/misc.js" type="text/javascript"></script>
</head>
<body>
  <div class="hero_area">
    <!-- header section strats -->
    <header class="header_section">
      <div class="container-fluid">
        <nav class="navbar navbar-expand-lg custom_nav-container ">
          <a class="navbar-brand" href="/index.html">
            <span>
            </span>
          </a>
          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>

          <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <div class="d-flex mx-auto flex-column flex-lg-row align-items-center">
              <ul class="navbar-nav  ">
              <li class="nav-item active">
                  <a class="nav-link" href="/index.html">Home <span class="sr-only">(current)</span></a>
                </li>
                <li class="nav-item active">
                  <a class="nav-link" href="/about.html">About <span class="sr-only">(current)</span></a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/contacts.html">Contact </a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/products.html">Products </a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/services.html">Services</a>
                </li>
		<li class="nav-item">
                  <a class="nav-link" href="/technology.html">Technology</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/news.html">News </a>
                </li>
		<li class="nav-item">
                  <a class="nav-link" href="/training.html">Events</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/help.html">Support </a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/download.html">Downloads</a>
                </li>
              </ul>
            </div>
            </div>
          </div>
        </nav>
      </div>
    </header>
    <!-- end header section -->
     <div style="margin:20px"><table border="0" align="center" width="100%" cellpadding="0" cellspacing="0"><tr><td width="80%" align="left" valign="bottom"><a name="technology"></a><h1>Molsoft's Technology</h1></td></tr></table><br /><p>
<a href="success-stories.html">Success Stories</a> | <a href="publications.html">Publications</a>
<p>
<div align=justify>
Molsoft develops new technology and proprietary algorithms for molecular
modeling with applications to protein and small molecule structure prediction,
docking and structure based drug design; molecular visualization and animation,
bioinformatics, cheminformatics, and laboratory information management systems.
</div>
<p>
<p>
<p>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech1s.png" height="235" width="230"></td>
<td>
<h2>Internal Coordinate Mechanics (ICM)</h2> MolSoft's ICM software package is based on the internal coordinates (IC) representation of molecular objects that naturally reflects the covalent bond geometry of molecules (Abagyan et al., 1994).  Unlike simple Cartesian coordinates, IC variables consist of covalent bond lengths and angles, torsion angles and six positional coordinates of a molecular object.  Because of chemical bond rigidity, most molecular objects can be accurately represented by free torsion variables while keeping covalent bond coordinates fixed.  This dramatically reduces the number of free variables in the system without sacrificing accuracy, while improving convergence time for conformational optimizations at least 1000-fold.  Moreover, further reduction of free variable space and system complexity in ICM can be achieved by effectively freezing IC variables in more rigid or less important parts of the model.  When carefully applied and validated, such complexity reductions reduce unnecessary noise in the modeling system and enable faster and more reproducible energy optimizations.
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech2s.png" height="198" width="230"></td>
<td>
<h2>Biased Probability Monte Carlo</h2> The core technology used in most of our structure prediction algorithms is
global free energy optimization in a subset of internal coordinates that
describes inter or inter-molecular geometry. For structure prediction and large
scale conformational sampling ICM employs a family of new global optimization
techniques such as: Biased Probability Monte Carlo (Abagyan and Totrov, 1994), pseudo-Brownian docking algorithm (Abagyan et al., 1994) and local deformation loop movements <a href="references.html#abm89">(Abagyan and Mazur, 1989 )</a>.
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech3s.png" height="209" width="230"></td>
<td>
<h2>Docking and Virtual Ligand Screening</h2> <a href="icm_pro.html">Docking (in ICM-Pro)</a> and <a href="vls.html">Screening (in ICM-Pro + VLS)</a> provides a unique set of tools for accurate ligand-protein docking,
peptide-protein docking, and protein-protein docking. The <a href="icm_pro.html">ICM-Pro</a> desktop modeling GUI interface offers a step-by-step docking menu or can be scripted for <a href="icm-cloud.html">large-scale</a> docking and screening.
<a href="docking.html">More...</a>
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech4s.png" height="178" width="230"></td>
<td>
<h2>3D Ligand Editor</h2> The ICM Ligand Editor is an intuitive graphical interface for ligand optimization and drug design. The editor was developed in close collaboration with Medicinal Chemists at Novartis and designed for ease of use and high accuracy ligand modeling. The ligand editor is available in <a href="icm_pro.html">ICM-Pro</a> and <a href="icm-chemist-pro.html">ICM-Chemist-Pro</a>.
<a href="ligand-editor.html">More...</a> 
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech5s.png" height="60" width="230"></td>
<td>
<h2>Novartis FOCUS</h2> MolSoft and Novartis have developed a new desktop modeling and communication environment for drug discovery called FOCUS based on MolSoft's Internal Coordinate Mechanics (ICM) software. The FOCUS platform is described in a publication in the ACS Journal of Chemical Information and Modeling (see <a href="http://pubs.acs.org/doi/abs/10.1021/ci500598e">Stiefl et al 2015</a>). FOCUS is a platform that helps users communicate chemical and structural data, develop new ideas and support decision making during the drug design cycle. FOCUS can be integrated into an informatics and high performance computing environment giving the user a single interface to many capabilities.
<a href="novartis-focus.html">More...</a>
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech6s.png" height="153" width="230"></td>
<td>
<h2>Induced Fit Docking</h2> ICM contains a selection of tools to account for pocket flexibility (induced fit) in docking and virtual screening. The importance of considering flexibility in proteins is well understood (1-3) and is an important consideration when undertaking structure-based drug design. There are three main approaches in ICM for incorporating induced fit:
<a href="induced-fit.html">More...</a>
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech7s.png" height="118" width="230"></td>
<td>
<h2>Atomic Property Fields</h2> The Atomic Property Field (APF) method developed by MolSoft ( Totrov 2008) is a 3D pharmacophoric potential implemented on a continuously distributed grid which can be used for ligand docking and scoring. APF can be generated from one or more high affinity scaffolds and seven properties are assigned from empiric physico-chemical components. 
<a href="apf.html">More...</a> 
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech8s.png" height="153" width="230"></td>
<td>
<h2>Rapid Isostere Discovery Engine (RIDE)</h2> <b>RIDE</b> is a fast 3D molecular similarity search method based on <a href="apf.html">Atomic Property Fields</a>, developed at MolSoft. RIDE searches databases of compound conformers for molecules that are isosteric to the query, i.e. have similar 3D configurations and distributions of atomic properties.RIDE is available in the ICM-Pro + <a href="vls.html">VLS</a> package. 
<a href="RIDE.html">More...</a>
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech9ss.png" height="229" width="233"></td>
<td>
<h2>PROTAC Modeling</h2> <b>Targeted Protein Degradation (TPD)</b> is an approach that is attracting substantial interest for modulating challenging drug
 targets. A major class of TPDs are <b>Proteolysis-Targeting Chimera protein degraders (PROTACs)</b>. <a href="https://www.pnas.org/doi/10.1073/pnas.141230798">PROTACs</a> are heterobifunctional molecules where two ligands are joined by linker. One ligand recruits the target and the other recruits and binds an E3 ubiquitin ligase. This interaction induces ubiquitylation of the target and degradation by the ubiquitin-proteasome system, the PROTAC is then recycled.
<a href="PROTAC.html">More...</a>
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech10s.png" height="182" width="230"></td>
<td>
<h2>RNA Drug Discovery</h2> 
MolSoft has a long history of developing methods for RNA drug discovery. The first ever successful virtual screen was against an RNA-protein interaction (tat-TAR). RNA drug discovery companies such as Arrakis Therapeutics, Novartis and Nymirum have published success stories using MolSoft ICM. <a href="rna.html">More...</a> 
</td>
</tr>
</table>
<hr>
<table border="0" cellpadding="10">
<tr> 
<td> <img src="images/tech11s.png" height="121" width="230"></td>
<td>
<h2>ICM Cloud</h2>
All ICM products can be run on the cloud - AWS, Google Cloud or Azure. You can read about the World's Largest ever virtual screen resulting in 3 new lead compounds from Novartis and a 680 million structure-based screen performed in 24 hours by USC. <a href="icm-cloud.html">More...</a>
</td>
</tr>
</table>
<hr>
<h2>Structure-based Prioritization of Protein Targets</h2>
<p>
<div align=justify>
The icmPocketFinder procedure identifies the substrate binding pockets in 98%
of all the cases (tested on over 10,000 pockets) An et al 2004). This procedure is based on
calculating the drug-binding density field and contouring it at a certain
level. In 2001 we published a fast procedure for accurate electrostatic
calculation using the boundary element algorithm (Totrov and Abagyan 2001). A combination of
"pocket-density" with other physical properties such as electrostatic
potential, hydrophobicity, hydrogen bonds is used to evaluate if a particular
protein target or protein-protein interface is "drugable" and prioritize the
targets. We developed a special procedure to improve the pocket models by
co-optimization of flexible pockets with some of the know ligands.  
</div>
<p>
<h2>Global Optimization of Compound Geometries</h2>
<p>
<div align=justify>
In addition to an internal coordinate force field, Molsoft-ICM platform allows
to perform global optimization and analysis of small molecule geometries by
performing free geometry optimization in Cartesian space using the MMFF94 force
field including fully automated atom type assignments. The conformational
generation procedure accumulates a non-redundant set of representative
molecular geometries.
</div>
<p>
<p>
<h2>Molsoft-ICM Scripting Language and Molecular Environment</h2>
<p>
<div align=justify>
Molsoft has developed more than several focused applications, we designed and
developed the whole computational environment for bioinformatics,
cheminformatics, protein modeling, protein design, docking and screening. The
environment is tied together by a common scripting language for molecules,
numbers, strings, vectors, matrices, tables, sequences, alignments, profiles
and maps This environment covers molecular graphics and production of molecular
animations. 
</div>
<p>
</div>
 
<!-- footer section -->
  <section class="container-fluid footer_section">
<!-- info section -->
  <section class="info_section layout_padding">
    <div class="container">
<a href="/sitemap.html" style="color:white;">Site Map</a> | <a href="https://www.molsoft.com/evaluation.html" style="color:white;">Request a Trial</a> | <a href="https://www.molsoft.com/new-features.html" style="color:white;">What's New</a> | <a href="https://groups.google.com/g/molsoft-icm-forum" style="color:white;">Knowledge Base</a>  | <a href="https://www.youtube.com/user/MolSoftHelp" style="color:white;">Videos</a> <br>
<br>
      <div class="info_social">
        <div>
          <a href="https://www.facebook.com/Molsoft/">
            <img src="/images/fb.png" alt="">
          </a>
        </div>
        <div>
          <a href="https://twitter.com/molsoft">
            <img src="/images/twitter.png" alt="">
          </a>
        </div>
        <div>
          <a href="https://www.linkedin.com/company/3634260/admin/">
            <img src="/images/linkedin.png" alt="">
          </a>
        </div>
        <div>
          <a href="">
            <img src="/images/insta.png" alt="">
          </a>
        </div>
      </div>
      <div>
        <p>
        </p>
      </div>
    </div>
  </section>

  <!-- end info_section -->

<p>
      &copy; 2024 All Rights Reserved MolSoft LLC

      <a href="/https://www.molsoft.com/terms-of-use.html"> Terms of Use | </a>

      <a href="/https://www.molsoft.com/privacy-policy.html"> Privacy Policy</a>
    </p>
  </section>
  <!-- footer section -->

  <script type="text/javascript" src="https://molsoft.com/js/jquery-3.4.1.min.js"></script>
  <script type="text/javascript" src="https://molsoft.com/js/bootstrap.js"></script>

</body>
</body>

</html>


