<html>
<head>
<title>LOMETS</title>




<!-- ==AAA== ======beginning of head====== -->
<!--
  When you compose a webpage at the webserver, please have following in mind:
     1, please keep the following 4 tags unchanged:
            ==AAA==
            ==BBB==
            ==CCC==
            ==DDD==
         These tags will be used to update the frame of the webpages.
     2, the content between '==AAA==' and '==BBB==' and content between 
        '==CCC==' and '==DDD==' will be mandatorily rewritten by the system 
        program. You can put your own contents outside these two regions.
	Thus, your page should look like the following:
	   [your content]
            ==AAA==
 	      [system content]
            ==BBB==
	   [your content]
            ==CCC==
 	      [system content]
            ==DDD==
-->

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" href="/css/all.css" type="text/css" />
<script type="text/javascript" src="/css/minmax.js"></script>
<script type="text/javascript" src="/css/all.js"></script>
</head>
<body>
<!-- End of declaration -->

<div id="z_top">
<a href="https://zhanggroup.org/" id="z_lab_logo"><img alt="Zhang Lab" src="https://zhanggroup.org/image/logo_top.jpg" /></a>
<a href="http://www.umich.edu" id="z_um_logo"><img alt="University of Michigan" src="https://zhanggroup.org/image/wordmark_m_web.gif" /></a>
</div>

<div id="z_menu">
  <a class="z_menu_main" href="https://zhanggroup.org">Home</a>
  <a class="z_menu_main" href="https://zhanggroup.org/research">Research</a>
  <a class="z_menu_main" href="https://zhanggroup.org/COVID-19/">COVID-19</a>
  <a class="z_menu_main" href="https://zhanggroup.org/services">Services</a>
  <a class="z_menu_main" href="https://zhanggroup.org/papers">Publications</a>
  <a class="z_menu_main" href="https://zhanggroup.org/people">People</a>
  <a class="z_menu_main" href="https://medicine.umich.edu/dept/dcmb/education/course-descriptions/bioinf-528-structural-bioinformatics">Teaching</a>
  <!-- <a class="z_menu_main" href="https://zhanggroup.org/funding">Funding</a> -->
  <a class="z_menu_main" href="https://zhanggroup.org/job">Job Opening</a>
<!--  <a class="z_menu_main" href="https://zhanggroup.org/facility">Facilities</a> -->
  <a class="z_menu_main" href="https://zhanggroup.org/news">News</a>
  <a class="z_menu_main" href="https://zhanggroup.org/forum">Forum</a>
  <a class="z_menu_main" href="https://zhanggroup.org/labonly">Lab Only</a>
  <div class="z_clear"></div>
</div>

<div id="z_body">
  <div id="z_sidebar">
    <a id="z_sidebar_title" href="https://zhanggroup.org/services">Online Services</a>
    <p class="z_sidebar_menu z_sidebar_menu1">
    <a href="https://zhanggroup.org/I-TASSER">&#9679;<span>I-TASSER</span></a>
      <a href="https://zhanggroup.org/I-TASSER-MTD">&#9679;<span>I-TASSER-MTD</span></a>
      <a href="https://zhanggroup.org/C-I-TASSER">&#9679;<span>C-I-TASSER</span></a>
      <a href="https://zhanggroup.org/CR-I-TASSER">&#9679;<span>CR-I-TASSER</span></a>
      
      <a href="https://zhanggroup.org/QUARK">&#9679;<span>QUARK</span></a>
      <a href="https://zhanggroup.org/C-QUARK">&#9679;<span>C-QUARK</span></a>

      <a href="https://zhanggroup.org/DMFold">&#9679;<span>DMFold</span></a>          
      <a href="https://zhanggroup.org/LOMETS">&#9679;<span>LOMETS</span></a>
      <a href="https://zhanggroup.org/MUSTER">&#9679;<span>MUSTER</span></a>
      <a href="https://zhanggroup.org/CEthreader">&#9679;<span>CEthreader</span></a>
      <a href="https://zhanggroup.org/SEGMER">&#9679;<span>SEGMER</span></a>

      <a href="https://zhanggroup.org/DeepFold">&#9679;<span>DeepFold</span></a>
      <a href="https://zhanggroup.org/DeepFoldRNA">&#9679;<span>DeepFoldRNA</span></a>
      <a href="https://zhanggroup.org/FoldDesign">&#9679;<span>FoldDesign</span></a>
      
      <a href="https://zhanggroup.org/COFACTOR">&#9679;<span>COFACTOR</span></a>
      <a href="https://zhanggroup.org/COACH">&#9679;<span>COACH</span></a>
      <a href="https://zhanggroup.org/MetaGO">&#9679;<span>MetaGO</span></a>
      <a href="https://zhanggroup.org/TripletGO">&#9679;<span>TripletGO</span></a>
      <a href="https://zhanggroup.org/IonCom">&#9679;<span>IonCom</span></a>
      
      <a href="https://zhanggroup.org/FG-MD">&#9679;<span>FG-MD</span></a>
      <a href="https://zhanggroup.org/ModRefiner">&#9679;<span>ModRefiner</span></a>
      <a href="https://zhanggroup.org/REMO">&#9679;<span>REMO</span></a>
      
      <a href="https://zhanggroup.org/DEMO">&#9679;<span>DEMO</span></a>
      <a href="https://zhanggroup.org/DEMO-EM">&#9679;<span>DEMO-EM</span></a>
      
      <a href="https://zhanggroup.org/spring">&#9679;<span>SPRING</span></a>
      <a href="https://zhanggroup.org/COTH">&#9679;<span>COTH</span></a>
      
      <a href="https://zhanggroup.org/Threpp">&#9679;<span>Threpp</span></a>
      <a href="https://zhanggroup.org/PEPPI">&#9679;<span>PEPPI</span></a>
      <a href="https://zhanggroup.org/BSpred">&#9679;<span>BSpred</span></a>
      
      <a href="https://zhanggroup.org/ANGLOR">&#9679;<span>ANGLOR</span></a>
      
      <a href="https://zhanggroup.org/EDock">&#9679;<span>EDock</span></a>
      <a href="https://zhanggroup.org/BSP-SLIM">&#9679;<span>BSP-SLIM</span></a>
      
      <a href="https://zhanggroup.org/SAXSTER">&#9679;<span>SAXSTER</span></a>
      
      <a href="https://zhanggroup.org/FUpred">&#9679;<span>FUpred</span></a>
      <a href="https://zhanggroup.org/ThreaDom">&#9679;<span>ThreaDom</span></a>
      <a href="https://zhanggroup.org/ThreaDomEx">&#9679;<span>ThreaDomEx</span></a>
      
      <a href="https://zhanggroup.org/EvoDesign">&#9679;<span>EvoDesign</span></a>
      <a href="https://zhanggroup.org/BindProf">&#9679;<span>BindProf</span></a>
      <a href="https://zhanggroup.org/BindProfX">&#9679;<span>BindProfX</span></a>
      <a href="https://zhanggroup.org/SSIPe">&#9679;<span>SSIPe</span></a>

      <a href="https://zhanggroup.org/GPCR-I-TASSER">&#9679;<span class="z_narrow">GPCR-I-TASSER</span></a>
      <a href="https://zhanggroup.org/MAGELLAN">&#9679;<span>MAGELLAN</span></a>

      <a href="https://zhanggroup.org/ResQ">&#9679;<span>ResQ</span></a>
      
      <a href="https://zhanggroup.org/STRUM">&#9679;<span>STRUM</span></a>
      <a href="https://zhanggroup.org/DAMpred">&#9679;<span>DAMpred</span></a>
    </p>
    <p class="z_sidebar_menu z_sidebar_menu2">
      <a href="https://zhanggroup.org/TM-score">&#9679;<span>TM-score</span></a>
      <a href="https://zhanggroup.org/TM-align">&#9679;<span>TM-align</span></a>
      <a href="https://zhanggroup.org/US-align">&#9679;<span>US-align</span></a>
      <a href="https://zhanggroup.org/MM-align">&#9679;<span>MM-align</span></a>
      <a href="https://zhanggroup.org/RNA-align">&#9679;<span>RNA-align</span></a>
      
      <a href="https://zhanggroup.org/NW-align">&#9679;<span>NW-align</span></a>
      <a href="https://zhanggroup.org/LS-align">&#9679;<span>LS-align</span></a>
      
      <a href="https://zhanggroup.org/EDTSurf">&#9679;<span>EDTSurf</span></a>
      <a href="https://zhanggroup.org/MVP">&#9679;<span>MVP</span></a>
      <a href="https://zhanggroup.org/MVP-Fit">&#9679;<span>MVP-Fit</span></a>
      
      <a href="https://zhanggroup.org/SPICKER">&#9679;<span>SPICKER</span></a>
      <a href="https://zhanggroup.org/HAAD">&#9679;<span>HAAD</span></a>
      <a href="https://zhanggroup.org/PSSpred">&#9679;<span>PSSpred</span></a>
      <a href="https://zhanggroup.org/3DRobot">&#9679;<span>3DRobot</span></a>
      
      <a href="https://zhanggroup.org/MR-REX">&#9679;<span>MR-REX</span></a>
      <a href="https://zhanggroup.org/I-TASSER-MR">&#9679;<span>I-TASSER-MR</span></a>
      
      <a href="https://zhanggroup.org/SVMSEQ">&#9679;<span>SVMSEQ</span></a>
      <a href="https://zhanggroup.org/NeBcon">&#9679;<span>NeBcon</span></a>
      <a href="https://zhanggroup.org/ResPRE">&#9679;<span>ResPRE</span></a>
      <a href="https://zhanggroup.org/TripletRes">&#9679;<span>TripletRes</span></a>
      <a href="https://zhanggroup.org/DeepPotential">&#9679;<span>DeepPotential</span></a>
      
      <a href="https://zhanggroup.org/WDL-RF">&#9679;<span>WDL-RF</span></a>
      <a href="https://zhanggroup.org/ATPbind">&#9679;<span>ATPbind</span></a>
      
      <a href="https://zhanggroup.org/DockRMSD">&#9679;<span>DockRMSD</span></a>
      <a href="https://zhanggroup.org/DeepMSA">&#9679;<span>DeepMSA</span></a>
      <a href="https://zhanggroup.org/FASPR">&#9679;<span>FASPR</span></a>
      
      <a href="https://zhanggroup.org/EM-Refiner">&#9679;<span>EM-Refiner</span></a>
      <a href="https://zhanggroup.org/GPU-I-TASSER">&#9679;<span>GPU-I-TASSER</span></a>
    </p>
    <p class="z_sidebar_menu z_sidebar_menu1">
      <a href="https://zhanggroup.org/BioLiP">&#9679;<span>BioLiP</span></a>
      <a href="https://zhanggroup.org/Ecoli">&#9679;<span>E. coli</span></a>
      
      <a href="https://zhanggroup.org/GLASS">&#9679;<span>GLASS</span></a>
      <a href="https://zhanggroup.org/GPCR-HGmod">&#9679;<span>GPCR-HGmod</span></a>
      <a href="https://zhanggroup.org/GPCR-RD">&#9679;<span>GPCR-RD</span></a>
      <a href="https://zhanggroup.org/GPCR-EXP">&#9679;<span>GPCR-EXP</span></a>
      
      <a href="https://zhanggroup.org/Tara-3D">&#9679;<span>Tara-3D</span></a>
      <a href="https://zhanggroup.org/TM-fold">&#9679;<span>TM-fold</span></a>
      
      <a href="https://zhanggroup.org/decoys">&#9679;<span>DECOYS</span></a>
      <a href="https://zhanggroup.org/potential">&#9679;<span>POTENTIAL</span></a>
      <a href="https://zhanggroup.org/RW">&#9679;<span>RW/RWplus</span></a>
      <a href="https://zhanggroup.org/EvoEF">&#9679;<span>EvoEF</span></a>
      <a href="https://zhanggroup.org/HPSF">&#9679;<span>HPSF</span></a>
      <a href="https://zhanggroup.org/THE-DB">&#9679;<span>THE-DB</span></a>
      <a href="https://zhanggroup.org/ADDRESS">&#9679;<span>ADDRESS</span></a>
      <a href="https://zhanggroup.org/Alpaca-Antibody">&#9679;<span>Alpaca-Antibody</span></a>
      
      <a href="https://zhanggroup.org/casp7">&#9679;<span>CASP7</span></a>
      <a href="https://zhanggroup.org/casp8">&#9679;<span>CASP8</span></a>
      <a href="https://zhanggroup.org/casp9">&#9679;<span>CASP9</span></a>
      <a href="https://zhanggroup.org/casp10">&#9679;<span>CASP10</span></a>
      <a href="https://zhanggroup.org/casp11">&#9679;<span>CASP11</span></a>
      <a href="https://zhanggroup.org/casp12">&#9679;<span>CASP12</span></a>
      <a href="https://zhanggroup.org/casp13">&#9679;<span>CASP13</span></a>
      <a href="https://zhanggroup.org/casp14">&#9679;<span>CASP14</span></a>
    </p>
  </div>
  <div id="z_page">
<!-- ==BBB== ======ending of head======== -->


<!-------begin text ---------->

<center>
<img src="/image/logo/LOMETStitle.gif"><br>
</center>

<!--
<h3> <font color="red">
Due to power outage and webserver failure, 
the LOMETS is currently unavailable for use. 
We are working on restoring the system which will come back around the first week of March.
We apologize for the inconvenience that this may cause.
</font></h3>
-->

<!--
<font color="red">
The server will be unavailable between January 10th and 12th, 2022, due to the datacenter
 maintenance. You can submit jobs during this period，but the job will run after January 
 12th. We apologize for any inconvenience this will cause.
</font>
-->

<p style="text-align:justify">
LOMETS (Local Meta-Threading Server, version 3) is a next-generation meta-server approach 
to template-based protein structure prediction and structure-based function annotation. 
The new program integrates multiple deep learning-based threading methods 
(<a href="/CEthreader">CEthreader</a>, DisCovER, EigenThreader, Hybrid-CEthreader, MapAlign) 
and state-of-the-art profile-based programs (FFAS3D, HHpred, HHsearch, MRFsearch, <a href="/MUSTER">MUSTER</a>, SparksX).
For the first time, LOMETS3 is extended to handling multi-domain proteins by introducing the 
domain partition (<a href="/FUpred">FUpred</a> and <a href="/ThreaDom">ThreaDom</a>) and 
assembly (<a href="/DEMO">DEMO</a>) modules. 
It also introduces a new module for fast full-length model construction using a gradient-based optimization
program (<a href="/DeepFold">DeepFold</a>), which is guided by restraints from 
deep-learning (<a href="/DeepPotential">DeepPotential</a>) and LOMETS top templates. 
Protein functions in LOMETS3 are predicted by the modified <a href="/COFACTOR">COFACTOR</a> method, 
which adds the LOMETS3 threading templates associated with structure analogs as templates 
in COFACTOR structure-based pipelines. 
Large-scaled benchmark tests showed that the overall template-recognition and full-length 
model construction accuracy is significantly beyond its predecessors (LOMETS and LOMETS2), 
due to the integration of deep-learning and multi-domain threading techniques.
LOMETS3 participated in 
<a href=https://www.predictioncenter.org/casp14/zscores_final.cgi?gr_type=server_only>CASP14</a>
as 'Zhang-TBM' and was ranked as one of the top methods for automatic protein structure prediction.
A detailed description of the LOMETS3 server can be seen on 
<a href=help.html>the About LOMETS</a> page.
The output model of LOMETS server is given by both PDB format and <a href="https://www.wwpdb.org/task/modelcif">ModelCIF</a> format
</a><img src="http://zhanglab.dcmb.med.umich.edu/image/new.gif"> now.
Please post your questions and comments about LOMETS at the
<a href=/forum>
Service System Discussion Board</a>. 
<p>

<b>The output</b> of the LOMETS server includes (<a href=example/>Example output</a>): 
<ul>
<li> Secondary structure prediction.
<li> Solvent accessibility prediction.
<li> Contact-map and distance-map prediction by <a href="/DeepPotential">DeepPotential</a>.
<li> Domain partition results shown in contact-map (optional).
<li> Individual domain threading/modeling results by LOMETS (optional).
<li> The best ten assembled templates from individual domain threading templates (optional).
<li> The best ten initial threading templates selected from 110 (=11x10) templates.
<li> Full-length models built by an L-BFGS system (<a href="/DeepFold">DeepFold</a>) using distance restraints from <a href="/DeepPotential">DeepPotential</a> and LOMETS templates.
<li> The best ten similar structure identified by TM-align using the first LOMETS model, and the associated functional annotations.
<li> Functional annotations (Gene Ontology term, Enzyme Commission number, and Ligand Binding residues) derived from top-ranking threading templates.
<li> Function predictions by a modified <a href="/COFACTOR">COFACTOR</a> method using LOMETS3 threading templates and structure analogs in COFACTOR structure-based pipelines. 
</ul>
<p>

<!--
<p>
<b>Tips</b> for modeling multi-domain proteins:
    <ul>
    <li>If you know the protein is a single-domain target or is a homologous target that has good templates that can cover most of the sequence, 
        you can select "run the protein as one part" in "Advanced options" section below the input box, then the system will ignore the 
        domain partition and assembly modules which can make your job complete slightly faster.</li>    
    </ul>
</p>
-->

<!--p>
<b>Tips</b> for modeling multi-domain proteins which are usually have >500 AA:
    <ol>
    <li>Use <a href="/FUpred">FUpred</a>, <a href="/ThreaDom">ThreaDom</a> or <a href="/ThreaDomEx">ThreaDomEx</a> servers to predict the domain boundary for your sequence, and
           then split the full-length sequence into domain-level sequences by FUpred, ThreaDom or ThreaDomEx domain partition information.</li>
    <li>Submit the domain-level sequences to LOMETS server to get the domain-level models, respectively.</li>
    <li>Submit the full-length sequence and domain-level models to <a href="/DEMO">DEMO</a> server, which will automatically assemble the full-length model.</li>
    </ol>
(You can also directly submit the full-length sequence to LOMETS server to get the model. Next, manually determine domain 
partitions based on the structure model of LOMETS as structural models generally show clear domain boundaries. 
Finally, repeat Step 2 and 3.)
    
</p-->

<br>
<center>
[<a href="example/index.html"><b>Example output</b></a>]
&nbsp
[<a href="help.html"><b>About LOMETS</b></a>]    
&nbsp
[<a href="/forum"><b>Forum</b></a>]
&nbsp
[<a href="check.html"><b>Check Previous Jobs</b></a>]
&nbsp
[<a href="benchmark.zip"><b>Benchmark datasets</b></a>]
</center>
<hr>
   
<h3>
LOMETS On-line
(<a href=example/>Example output</a>) 
</h3>
<ul>
<FORM METHOD="POST" id="FORM1" name="FORM1" ENCTYPE="MULTIPART/FORM-DATA" ACTION="bin/receive.cgi">
<p>
<h3> <font color="red">
</font></h3>
Cut and paste your sequence here (&lt;1500 residues in <a href=/FASTA>FASTA format</a>):
<a href="javascript:example()">Example input</a><br>
<TEXTAREA NAME="SEQUENCE" ROWS=6 COLS=80></TEXTAREA>
<p>
Or upload the sequence from your local computer:
<br>
<input type="file" name="seq_file" size="50">
<br>
Email: (not required but recommended, where results will be sent to)<br>
<input type="text" name="REPLY-E-MAIL" size=30>
<br>
ID: (optional, your given name of the protein)<br>
<input type="text" name="TARGET" size=30>
<br>
    <br>
    <a id="option1" href="javascript:formAction('show',1)">
    <img id="optionGraphic1" border=0 src="img/right_triangle.gif"/>

    <b>Advanced options</b></a>
    </a>
    <div id="optionForm1">

    <ul>
    <li><b>Exclude templates:</b>
    <br><input type="radio" name="HOMOFLAG" value="real" checked>keep all identified templates (recommended)
    <br><input type="radio" name="HOMOFLAG" value="benchmark">remove templates sharing &gt;30% sequence identity with target
    </ul>

    <ul>
    <li><b>Automatic domain partition and assembly:</b>
    <br><input type="radio" name="DOMAINFLAG" value="true" checked>run domain partition and assembly 
    (recommended, see <a href=help.html#DOMAINFLAG>details</a>)
    <br><input type="radio" name="DOMAINFLAG" value="false">run threading without domain partition 
    (if you believe your protein is of single-domain)
    </ul>

    <ul>
    <li><b>Function prediction by COFACTOR:</b>
    <br><input type="checkbox" name="PREFUNC" value="true" checked>predict protein function (need additional 1-10 hours).<p>
    </ul>

    </div>
<INPUT TYPE="submit" VALUE="Run LOMETS"> <INPUT TYPE="reset" VALUE="Clear form">
<br>
<br>
See server response time 
<a href=help.html#runtime>here</a>
</FORM>
</ul>
<br>
<hr width=30% align=left>
<b>LOMETS Resource:</b>
<ul>
<a href=https://www.xsede.org><img src=img/power_by_xsede.gif></a>
</ul>

<!--b>LOMETS Download:</b>
<ul>

<!--li><a href=benchmark2.html>Download the benchmark set of 614 non-redundant
proteins used by LOMETS</a -->

<!--li>
<a href=https://zhanglab.dcmb.med.umich.edu/C-I-TASSER/download.html>Download
LOMETS Package through C-I-TASSER Suite</a>
</ul-->

<p>
<hr width=30% align=left>
<b>References</b>:
<ul>

<li>
Wei Zheng, Qiqige Wuyun, Xiaogen Zhou, Yang Li, Peter Freddolino, Yang Zhang.
LOMETS3: Integrating deep-learning and profile-alignment for advanced protein template recognition and function annotation.
Nucleic Acids Research, 50: W454-W464 (2022)
[<a href=https://academic.oup.com/nar/article/50/W1/W454/6568492>PDF of manuscript</a>].
</li>
    
<li>
Wei Zheng, Chengxin Zhang, Qiqige Wuyun, Robin Pearce, Yang Li, Yang Zhang.
LOMETS2: improved meta-threading server for fold-recognition and structure-based 
function annotation for distant-homology proteins.
Nucleic Acids Research, 47: W429-W436 (2019)
[<a href=https://zhanglab.dcmb.med.umich.edu/papers/2019_9.pdf>PDF of manuscript</a>]
[<a href=https://zhanglab.dcmb.med.umich.edu/papers/2019_9_support.pdf>PDF of Support Information</a>].

<li>Sitao Wu, Yang Zhang. 
LOMETS: A local meta-threading-server for protein structure prediction.
Nucleic Acids Research, 35: 3375-3382 (2007)
[<a href="/papers/2007_1.pdf">PDF of manuscript</a>].

</ul>
<p>

<!--------end text -------->

<!-- ==CCC== ======beginning of tail==== -->
 </div></div> <!-- for closing all the opening tags. -->
</div>
<p>
  
<p id="z_foot">
<a href="mailto:yangzhanglab\@umich.edu">yangzhanglab<img src=/image/at.gif width=10 border=0>umich.edu</a>  
|  (734) 647-1549 | 100 Washtenaw Avenue, Ann Arbor, MI 48109-2218
</p>
<br>
</body>
</html>
<!-- ==DDD== =======ending of tail====== -->


<script type="text/javascript">

function example()
{
    var STR=IO("example/seq.fasta");
    document.FORM1.SEQUENCE.value = STR;
}

function IO(U, V) 
{
    var X = !window.XMLHttpRequest ? new ActiveXObject('Microsoft.XMLHTTP') : new XMLHttpRequest();
    X.open(V ? 'PUT' : 'GET', U, false );
    X.setRequestHeader('Content-Type', 'text/html');
    X.send(V ? V : '');
return X.responseText;
}

function formAction (action, id)
{
    if (action == 'show')
    {
        document.getElementById("optionForm"+id).style.display= "block";
        document.getElementById("optionGraphic"+id).src="img/down_triangle.gif";
        document.getElementById("option"+id).href= "javascript:formAction('hide', "+id+")";
    }

    if (action == 'hide')
    {
        document.getElementById("optionForm"+id).style.display= "none";
        document.getElementById("optionGraphic"+id).src="img/right_triangle.gif";
        document.getElementById("option"+id).href= "javascript:formAction('show', "+id+")";
    }
}
</script>
